src/app.rb

```diff
@@ -1,15 +1,23 @@
-require './src/person'
-require './src/trimmer_name'
-require './src/capitalize_name'
-require './src/student'
-require './src/classroom'
-require './src/book'
-require './src/teacher'
+require_relative 'person'
+require_relative 'trimmer_name'
+require_relative 'capitalize_name'
+require_relative 'student'
+require_relative 'classroom'
+require_relative 'book'
+require_relative 'teacher'
+require_relative 'storage'
 
 class App
   $people_list = []
   $book_list = []
   $rental_list = []
+  $storage = Storage.new
+
+  def read_from_storage
+    $people_list = $storage.persons_from_map
+    $book_list = $storage.books_from_map
```

Comment:
- [ ] The app fails when I tried to list all books from a file that doesn't exist. It will be nice to handle the case when one or more of the files are missing (the app should work and show no errors)

This particular error comes from the `list_books` method. My marker is on `line 18` because there are no active recent changes on the `list_books` method.

Screenshot  |
:----------------:|
<img width="679" alt="Screen Shot 2022-11-08 at 11 10 50 PM" src="https://user-images.githubusercontent.com/39523312/200696792-9b4e54b3-e821-48c5-b554-d728388e3657.png"> |